cwms_radar_api/src/main/java/cwms/radar/ApiServlet.java

```diff
@@ -102,6 +102,7 @@ public class ApiServlet extends HttpServlet {
     public static final String OFFICE_ID = "office_id";
     public static final String DATA_SOURCE = "data_source";
     public static final String DATABASE = "database";
+    public static final String VERSION = "3.0";
```

Comment:
why are we jumping to 3?

Reply:
It doesn't have to be 3.0.  2.1?

Reply:
shove the gradle version property into a resource file and pull it from there?

Reply:
oh. 2.4 to match the gradle.

Reply:
or at least the major/minor. There's a use case to know if it's def newer to run performance checks but I can add a hash laster.